src/shared/mq_op.c

```diff
@@ -50,6 +50,22 @@ int StartMQ(const char *path, short int type, short int n_attempts)
     }
 }
 
+/* Reconnect to message queue */
+int MQReconnectPredicated(const char *path, bool (*fn_ptr)()) {
+    int rc = 0;
+    while ((rc = OS_ConnectUnixDomain(path, SOCK_DGRAM, OS_MAXSTR + 256)), rc < 0){
+        if ((*fn_ptr)()) {
+            return OS_INVALID;
+        }
+        merror("Can't connect to '%s': %s (%d).", path, strerror(errno), errno);
```

Comment:
Unable to connecto to

Reply:
I think is more appropriate to add this message to error_messages.h, and also prepending a numerical id